library/src/main/java/com/novoda/downloadmanager/lib/DownloadService.java

```diff
@@ -345,7 +345,7 @@ private boolean updateLocked() {
                 continue;
             }
 
-            if (!isActive && downloadReadyChecker.canDownload(downloadBatch)) {
+            if (downloadReadyChecker.canDownload(downloadBatch) && !isActive) {
```

Comment:
:x: I'm still not sold on this, it doesn't make sense to check if inactive downloads `canDownload`, we'll end up spamming `canDownload` for no reason which will lead to other problems for example:

if a client has a space check in `canDownload` and has an active download which the device has enough space for with other paused items which the device does not have space for, the library will be incorrectly checking against the paused items and if the logic to handle out of space errors is inside the client side `canDownload` it'll be triggered even though there is enough space for the active download